person.rb

```diff
@@ -0,0 +1,38 @@
+# Create a class person
```

Comment:
```ruby

- [ ] instance variables should be able to be modified that's why they are defined as attr_accessors.

  attr_accessor :name, :age 
  

- [ ]  *:id should only be able to be retrieved so only the Getter is defined

  attr_reader :id 
```

Reply:
Hello! @arslanbisharat, Kindly did you read the project requirements?